wikimedia/wikipedia-ios
# Batch 2 - change summary tags to edit tags

**Phabricator:** https://phabricator.wikimedia.org/T366731

### Notes
This PR changes our edit summary hashtags over to edit tags when editing articles and talk pages.

### Test Steps
_Note: This is listed here for completion, but we can lean more on QA for all of these._
1. Edit an article section from an edit pencil on Test Wiki. Confirm edit tag on Desktop shows "App section source".
2. Edit an article's full source from the over flow menu on Test Wiki. Confirm edit tag on Desktop shows "App full source".
3. Edit an article by highlighting some text, then tapping "Edit" in the context menu on Test Wiki. Confirm edit tag on Desktop shows "App select source".
4. Edit an article talk page by tapping "Edit source" in the overflow menu on a Test Wiki article talk page. Confirm edit tag on Desktop shows "App talk source".
5. Post a new topic in a talk page by tapping the + button on a talk page on Test Wiki. Confirm edit tag on Desktop shows "App talk topic".
6. Post an inline reply in a talk page on Test Wiki. Confirm edit tag on Desktop shows "App talk reply".

Confirm that all of these no longer edits have a hashtag appended to the edit summary.
